Sources/NIOKit/NIOKitTestCase.swift

```diff
@@ -0,0 +1,35 @@
+import NIO
+import XCTest
+
+protocol NIOKitTestCaseProt: class {
+    var group: EventLoopGroup! { get set }
+}
+
+extension NIOKitTestCaseProt {
```

Comment:
```suggestion
extension NIOKitTestCaseProtocol {
```
I think that better lines up w/ Swift style.

Reply:
Agreed. I’ll fix it 😊